framework-cloud/framework-openshift-operator/src/main/java/org/kie/cloud/openshift/operator/scenario/OpenShiftOperatorScenario.java

```diff
@@ -172,6 +172,22 @@ protected void removeConfigurationFromExternalDeployment(ExternalDeployment<?, ?
         ((ExternalDeploymentOperator) externalDeployment).removeConfiguration(kieApp);
     }
 
+    protected String getLatestOperatorVersion() {
+        String operatorImageTag = OpenShiftOperatorConstants.getKieOperatorImageTag();
+        String[] split = operatorImageTag.split(":");
+        ImageStream operatorImageStream = project.getOpenShiftAdmin().getImageStream(split[0]);
+        if (Objects.nonNull(operatorImageStream)) {
+            final String streamTag = split.length > 1 ? split[1] : "latest";
+            operatorImageTag = operatorImageStream.getStatus().getDockerImageRepository() + ":" + streamTag;
+        }
+
+        return operatorImageTag;
+    }
+
+    protected String overridesVersionTag() {
```

Comment:
Can you please add some comment for this method. It can looks like that it is actually not needed as it is result null. But this can be override in scenario.

Reply:
done